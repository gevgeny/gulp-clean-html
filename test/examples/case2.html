<div>
    <div some-attr="foo">
        node1
    </div>
    <div to-remove="foo">
        node2
    </div>
    <div some-attr="bar">
        node3
    </div>
    <div to-remove="bar">
        node4
    </div>
</div>