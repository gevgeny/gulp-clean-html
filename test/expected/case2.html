<div>
    <div some-attr="foo">
        node1
    </div>
    
    <div some-attr="bar">
        node3
    </div>
    
</div>